Maple2.Database/sql/system_banner.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `game-server`.`system-banner` (Name,`Type`,`Function`,FunctionParameter,Url,`Language`,BeginTime,EndTime) VALUES
+	 ('homeproduct_emulator',1,1,'','https://i.imgur.com/TV47C2m.png',-1,'2010-01-01 00:00:00','2030-01-01 00:00:00');
```

Comment:
how about hosting this image on `Maple2.Server.Web`?

Reply:
What do you have in mind? I wasn't thinking of putting it in ugc-resources because it's not ugc. Banners also require the full url

Reply:
`Maple2.Server.Web` is essentially just a website, so you could host anything on it.